src/components/ImportForm/SampleSection/SampleInfoAlert.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { Alert, AlertActionCloseButton } from '@patternfly/react-core';
+import { useLocalStorage } from '../../../hooks';
+
+const SAMPLES_FLOW_INSTRUCTION_KEY = 'samples-flow-instruction';
+
+type SampleInfoAlertProps = {
+  localStorageKey: string;
+};
```

Comment:
I feel like both alerts should share the same key as they both show the same messaging within the same flow.

Reply:
sure, will keep one key for both alerts.  Initially I was thinking about having one key then closing the first one will close the second one before even user the seeing it but like you said its same message within same flow it makes sense now.